src/dbnode/persist/fs/seek_manager.go

```diff
@@ -496,6 +500,51 @@ func (m *seekerManager) updateOpenLeaseHotSwapSeekers(
 	return wg, updateOpenLeaseResult, nil
 }
 
+func (m *seekerManager) RelinquishShard(
+	namespace ident.ID,
+	shard uint32,
+) error {
+	if !m.namespace.Equal(namespace) {
+		return fmt.Errorf("seekerManager's namespace %q != provided namespace %q",
```

Comment:
The `SeekerManager` will receive updates for all namespace so this should be a no-op instead of an error since it will happen all the time

Reply:
I removed the check since we're not having `namespace` as an argument anymore.